action.yml

```diff
@@ -7,12 +7,22 @@ inputs:
 
       By default, the `github.token` is configured. See https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication#permissions-for-the-github_token
       for a description of the default token's permissions.
+  extra_nix_config:
+    description: |
+      Additional configuration text to append to nix.conf. See https://docs.lix.systems/manual/lix/2.90/command-ref/conf-file.html
+      for a description of the config file format.
 runs:
   using: "composite"
   steps:
+    - name: Cache installer
+      uses: actions/cache@v4
+      with:
+        path: /tmp/lix-installer
+        key: lix-installer-${{ runner.os }}-${{ runner.arch }}
```

Comment:
Unless you really wanted to expand the scope of this PR, don't do anything about the following, but here are some thoughts I had, which does not prevent this from being merged.

 - https://github.com/samueldr/lix-gha-installer-action/issues/9